ui/public/config.json

```diff
@@ -1,5 +1,6 @@
 {
-    "url": "https://172.21.254.13:6443",
-    "url_salt": "http://172.21.254.13:4507",
-    "url_prometheus": "http://172.21.254.46:31266"
+    "url": "https://{ip}:8443/api/kubernetes",
+    "url_salt": "https://{ip}:8443/api/salt",
+    "url_prometheus": "https://{ip}:8443/api/prometheus",
+    "url_grafana": "https://{ip}:8443/grafana"
```

Comment:
I think those paths are intended to be exposed (via Ingress) on the same host as the UI itself, so maybe using:

```
{
  "url": "/api/kubernetes",
  "url_salt": "/api/salt",
  "url_prometheus": "/api/prometheus",
  "url_grafana": "/grafana",
}
```

is acceptable, even locally. No?

Reply:
If the goal is to pass some `INGRESS_HOST` when running the UI from your laptop, maybe that's something to add in some `npm` script when running a dev server?

Reply:
The goal is to be able to hook the local development UI (accessible at localhost:3000) to any backend (cluster) which serves as a mock. This local development UI is not part of the cluster (so Ingress). We can discuss it IRL ;)